packages/notebook/src/panel.ts

```diff
@@ -128,6 +135,28 @@ export class NotebookPanel extends DocumentWidget<Notebook, INotebookModel> {
     super.dispose();
   }
 
+  /**
+   * Returns the URL converting this notebook to a certain
+   * format with nbconvert.
+   *
+   * If the `baseUrl` was not passed into the panel, then
+   * undefined is returned.
+   */
+  getNBConvertURL(
```

Comment:
I wonder if this function might fit a little better in the `actions` namespace. I'm not sure, but something is bothering me about having it on the `Panel`...

Reply:
I put it in the `PageConfig`, where I think it fits much better.